components/collector/src/source_collectors/sonarqube/security_warnings.py

```diff
@@ -75,8 +70,16 @@ async def _parse_source_responses(self, responses: SourceResponses) -> SourceMea
             entities=vulnerabilities.entities + hotspots,
         )
 
-    async def __entity(self, hotspot) -> Entity:
-        """Create the security warning entity."""
+    def __include_hotspot(self, hotspot) -> bool:
+        """Return whether to include the hotspot."""
+        review_priorities = self._parameter("review_priorities")
+        review_priority = hotspot["vulnerabilityProbability"].lower()
```

Comment:
I had to read twice, but to be sure: we used to check on param.upper and now we check on value.lower so it's semantically the same. But that assumes that users have always typed the parameters in lowercase, no?

Reply:
No, the parameter is multiple choice, so there's no typing involved.